CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 # Change Log
 All notable changes to the "azure-appservice" extension will be documented in this file.
 
+## 0.9.0 - 2018-07-12
+### Added
+- Remote file editting for Web Apps and slots (previously in preview)
+- Log file access for Web Apps and slots
+- Deployment defaults for a project workspace. The web app deployed to and the subpath deployed will be saved and used for consequent deployments.
+- Support for WAR deploy
+
+### Changed
+- `Create New Web App...` has been redesigned to make smart defaults and have fewer prompts.
+    - If the previous creation experience is desired, change the VS Code user-setting `{appService.advancedCreation: true}`
+- Output window is no longer shown automatically.  Long running notifications have been implemented.
+- Minor text fixes.
```

Comment:
Remove 'minor text fixes' - doesn't seem like useful information

Reply:
It was a joke of this:
https://knowyourmeme.com/memes/pokemon-go-updates-controversy
I thought you'd get it!

Reply:
![over my head](https://media.giphy.com/media/wSCAy1zJbcUG4/giphy.gif)